SQLBuilder/Universal/Expr/BetweenExpr.php

```diff
@@ -23,7 +23,11 @@ public function __construct($exprStr, $min, $max) {
     }
 
     public function toSql(BaseDriver $driver, ArgumentArray $args) {
-        return $this->exprStr . ' BETWEEN ' . $driver->deflate($this->min, $args) . ' AND ' . $driver->deflate($this->max, $args);
+        $column = $this->exprStr;
```

Comment:
The left operand of "BETWEEN" may be an expr object or something that could be convert into SQL. 

I think we should add a "is_string" check here and rename $exprStr to $expr